Dockerfile

```diff
@@ -27,7 +28,9 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
+
 COPY --from=builder /workspace/manager .
+COPY embeds/ embeds/
```

Comment:
I syspect this should not be required since you're embedding everything in the binary.

Reply:
+1